pagopa/io-app
# fix: [#176660226] Fix iPhone 12 (Mini, Pro...) home button overlapping onto the tab bar

## List of changes proposed in this pull request
Before: 
<img width="200" alt="before" src="https://user-images.githubusercontent.com/26572302/107237807-6d99d900-6a27-11eb-84d2-d12ceebb2c80.png">

After:
<img width="200" alt="after" src="https://user-images.githubusercontent.com/26572302/107237832-74285080-6a27-11eb-98df-4cf5ae99c28e.png">

## How to test
Open the app with any iPhone of the iPhone 12 product line, see the tab bar not overlapping.
